ruby/vampires.rb

```diff
@@ -0,0 +1,59 @@
+puts "How many employees will be processed?" 
+employees = gets.chomp.to_i #convert for arithmetic
+
+while employees > 0 
+	puts "What is your name?" # Question 1
+	name = gets.chomp
+	puts "How old are you?"   # Question 2
+	age = gets.chomp.to_i
+	puts "What year were you born?" # Question 3
+	yearmatch = true
+	byear = gets.chomp.to_i
+		if (2016-byear) == age # roughly check age by using current year
+			yearmatch # yearmatch is true still
+		else

```

Comment:
There is some refactoring that could be done, but the conditional logic and logic flow is good.  For instance: 
```
if garlic_bread.downcase == "yes" || garlic_bread.downcase == "y"
    garlic_bread = true
else
   garlic_bread = false
end
```
can be refactored to:
```
garlic_bread = garlic_bread.downcase == "yes" || garlic_bread.downcase == "y"
```
If `garlic_bread.downcase` is neither of those answers, `garlic_bread` will be set to false.